charts/stable/stash/ci/ct-values.yaml

```diff
@@ -9,9 +9,13 @@ persistence:
     enabled: true
     emptyDir:
       enabled: true
+    size: 10M
     mountPath: /root/.stash
+    accessMode: ReadWriteOnce
```

Comment:
Why not use a `type: emptyDir`? this of course is the next best thing, but then the `emptyDir` key can be removed completely :)

Reply:
Not sure I understand, I was following how the paperless PR was handling it.

Reply:
I noticed... I wanted to comment / ask the question on that as well. In other charts that require some kind of persistence during the `ct install` CI phase, we just give it an emptyDir instead of a real PVC. That way we don't have to rely on a CSI being present. So it would become something like this:

```yaml
persistence:
  config:
    enabled: true
    type: emptyDir
```

Reply:
I spoke with @angelnu about this, and we can continue to use emptyDir's 🙂